RUM proxy configuration: add intake url for previous sdk version (#15087)
DataDog/documentation: content/en/real_user_monitoring/guide/proxy-rum-data.md

2. Forward the request to the URL set in the `ddforward` query parameter using the POST method.
4. The request body must remain unchanged.

**IMPORTANT**: Ensure the `ddforward` attribute points to a valid Datadog endpoint for your [Datadog site][2]. Failing to do so may expose your systems to SSRF/XSS attacks. The list of valid intake URL patterns per sites are:

> For latest SDK version:

Reviewer: <!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏Maybe use tabs to have a more succinct output ([syntax example](https://github.com/DataDog/browser-sdk/pull/1648/commits/1591aecb5e70abe72c7dfe01000258a42049dc6e#diff-a9c53af1f3e9e5b103b5bdef39b52eac991688984575a03139b6fcfdf143ecc5R40))

2. Forward the request to the URL set in the `ddforward` query parameter using the POST method.
4. The request body must remain unchanged.

> **IMPORTANT**: Ensure the `ddforward` attribute points to a valid Datadog endpoint for your [Datadog site][2]. Failing to do so may expose your systems to SSRF/XSS attacks. The list of valid intake URL patterns per sites are:

Reviewer 2: ```suggestion
Ensure the `ddforward` attribute points to a valid Datadog endpoint for your [Datadog site][2]. Failure to do so may expose your systems to SSRF/XSS attacks. 

The site parameter is an SDK [initialization parameter][3]. Valid intake URL patterns for each site are listed below:
```

{{< tabs >}}

> {{% tab "latest version" %}}

Reviewer 2: ```suggestion
{{% tab "Latest version" %}}
```

| Site    | Valid intake URL Pattern                       | Site Parameter      |
|---------|------------------------------------------------|---------------------|
| US1     | `https://*.browser-intake-datadoghq.com/*`     | `datadoghq.com`     |
| US3     | `https://*.browser-intake-us3-datadoghq.com/*` | `us3.datadoghq.com` |
| US5     | `https://*.browser-intake-us5-datadoghq.com/*` | `us5.datadoghq.com` |
| EU1     | `https://*.browser-intake-datadoghq.eu/*`      | `datadoghq.eu`      |
| US1-FED | `https://*.browser-intake-ddog-gov.com/*`      | `ddog-gov.com`      |

{{% /tab %}}

> {{% tab "before v4" %}}

Reviewer 2: ```suggestion
{{% tab "Before `v4`" %}}
```

| Site    | Valid intake URL Pattern             | Site Parameter      |
|---------|--------------------------------------|---------------------|
| US1     | `https://*.logs.datadoghq.com/*`     | `datadoghq.com`     |
| US3     | `https://*.logs.us3-datadoghq.com/*` | `us3.datadoghq.com` |
| EU1     | `https://*.logs.datadoghq.eu/*`      | `datadoghq.eu`      |
| US1-FED | `https://*.logs.ddog-gov.com/*`      | `ddog-gov.com`      |

{{% /tab %}}
{{< /tabs >}}

> **Note:** Site parameter is an SDK [initialization parameter][3].

Reviewer 2: ```suggestion
```
